src/modules/createInvolvementApi.js

```diff
@@ -0,0 +1,10 @@
+const api = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/';
+
+const createApp = async () => fetch(api, {
+  method: 'POST',
+}).then((response) => response.json())
+  .catch((error) => {
+    throw error;
+  });
```

Comment:
**Changes Required**
- [ ] Fair effort on using the `async await`  promise syntactic sugar. However, you're mixing two syntax here, and the `async` keyword in this context is redundant. May we please remove the `.then` and stick to the `async await` format.

Reply:
Changed ✔